tests/base/test_extractors.py

```diff
@@ -238,3 +252,14 @@ def test_spacy_ner_extractor(spacy_nlp):
         'value': 'West',
         'entity': 'LOC',
         'confidence': None}
+
+    # Test dimension filtering
+
+    example = Message("anywhere in the West", {
+        "intent": "restaurant_search",
+        "entities": [],
+        "spacy_doc": spacy_nlp("anywhere in the west")})
+    ext = SpacyEntityExtractor({"dimensions": ["EVENT"]})
```

Comment:
```suggestion
    _config = RasaNLUModelConfig({"pipeline": [{"name": "ner_spacy"}]})
    _config.set_component_attr("ner_spacy", dimensions=["EVENT"])
    ext = component_builder.create_component("ner_spacy", _config)
```

we should create the `SpacyEntityExtractor` through the config, since this is where we'll be setting the new `dimensions` filter.

Reply:
forgot to add that adding this suggestion and the one above will require also passing `create_component` to `test_spacy_ner_extractor()`